src/Models/Organization.php

```diff
@@ -5,9 +5,32 @@
 namespace Blumilk\Meetup\Core\Models;
 
 use Blumilk\Meetup\Core\Formats;
+use Illuminate\Database\Eloquent\Collection;
 use Illuminate\Database\Eloquent\Factories\HasFactory;
 use Illuminate\Database\Eloquent\Relations\HasMany;
+use Illuminate\Support\Carbon;
 
+/**
+ * @property int $id
+ * @property string $name
+ * @property string|null $description
+ * @property string $location
+ * @property string $organizationType
+ * @property string $foundationDate
+ * @property string $numberOfEmployers
+ * @property string $logo
+ * @property string|null $websiteUrl
+ * @property string|null $facebookUrl
+ * @property string|null $linkedinUrl
+ * @property string|null $instagramUrl
+ * @property string|null $youtubeUrl
+ * @property string|null $twitterUrl
+ * @property string|null $githubUrl
+ * @property Carbon|null $createdAt
+ * @property Carbon|null $updatedAt
+ * @property-read Collection|array<Meetup> $meetups
```

Comment:
```suggestion
 * @property-read Collection<Meetup> $meetups
```

This is not an array of meetups. Laravel since version 9 has generic templates for collections, so you should use it.